examples/ai-billing/app/analytics/page.tsx

```diff
@@ -0,0 +1,168 @@
+import { BarChart, BarList, Callout, Card, Col, Divider, Grid, Metric, Text } from "@tremor/react";
+import { Unkey } from "@unkey/api";
+import { FilterDateRange } from "./filter";
+const unkey = new Unkey({ rootKey: process.env.UNKEY_ROOT_KEY! });
+import { auth } from "@/auth";
+
+type Props = {
+  searchParams: {
+    keyId?: string;
+    ownerId?: string;
+    start?: string;
+    end?: string;
+  };
+};
+
+export default async function AnalyticsPage(props: Props) {
+  const sess = await auth();
+  const ownerId = sess?.user?.id;
+
+  const allKeys = await unkey.apis.listKeys({
+    apiId: process.env.UNKEY_API_ID!,
+    ownerId,
+  });
+  if (allKeys.error) {
+    return (
+      <Callout title={allKeys.error.message} color="red">
+        <pre>{JSON.stringify(allKeys.error, null, 2)}</pre>
+      </Callout>
+    );
+  }
+  if (props.searchParams.keyId) {
+    allKeys.result.keys = allKeys.result.keys.filter((k) => k.id === props.searchParams.keyId);
+  }
+  if (props.searchParams.ownerId) {
+    allKeys.result.keys = allKeys.result.keys.filter(
+      (k) => k.ownerId && k.ownerId === props.searchParams.ownerId,
+    );
+  }
+
+  const _keysByOwner = allKeys.result.keys.reduce((acc, key) => {
+    if (!key.ownerId) {
+      return acc;
+    }
+    if (!acc[key.ownerId]) {
+      acc[key.ownerId] = 0;
+    }
+    acc[key.ownerId] += 1;
+    return acc;
+  }, {} as Record<string, number>);
+
+  const remainingByOwner = allKeys.result.keys.reduce((acc, key) => {
+    if (!key.id) {
+      return acc;
+    }
+    if (!acc[key.id]) {
+      acc[key.id] = 0;
+    }
+    acc[key.id] += key.remaining || 0;
+    return acc;
+  }, {} as Record<string, number>);
+
+  const t = new Date();
+  const start = props.searchParams.start ? parseInt(props.searchParams.start) : t.getTime();
+  t.setUTCMonth(t.getUTCMonth() - 1);
+  const end = props.searchParams.end ? parseInt(props.searchParams.end) : Date.now();
+
+  // TODO: create a new API for this. parallel loading is not going to scale
+  const keys = await Promise.all(
```

Comment:
this could use the `/v1/keys.getVerifications` endpoint, right?

Reply:
cause we're getting individual key usage for all keys belonging to an owner, which is exactly what that endpoint does